Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "gleam"
 version = "0.0.1"
+license = "ASL2"
```

Comment:
Fyi, cargo uses http://spdx.org/licenses/ - it won't pass validation otherwise. I know you guys don't publish to cargo yet, but it is good to get into the habit. :)